cfe_internal/enterprise/federation/federation.cf

```diff
@@ -37,6 +37,13 @@ bundle agent config
       "am_on" expression => strcmp("on", "$(data[target_state])"),
         scope => "namespace";
 
+      # _stdlib_path_exists_getenforce and paths.getenforce are defined by masterfiles/lib/paths.cf
```

Comment:
I didn't really have to move this to config since vars and classes are always evaluated and available in their respective namespaces. It just feels more natural to put it here since it is a "shared" space.